src/index.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import "./css/bootstrap.min.css";
+import "./css/my-styles.css";
```

Comment:
Please use full path unless the file is not in the same folder

Reply:
Done